cli/tests/pcluster/test_utils.py

```diff
@@ -198,7 +198,7 @@ def test_init_from_instance_type(mocker, caplog):
     assert_that(c4_instance_info.gpu_count()).is_equal_to(0)
     assert_that(caplog.text).is_empty()
     assert_that(c4_instance_info.max_network_cards()).is_equal_to(1)
-    assert_that(c4_instance_info.network_cards_index_list()).is_equal_to([0])
+    assert_that(c4_instance_info.network_cards_list()[0].network_card_index()).is_equal_to(0)
```

Comment:
Maybe later we can add other instance typed like p4 and hpc6id for this unit test on Instance Info